### DISCUSSION TITLE
How to Retain elastic search data

### DISCUSSION THREAD

**Discussion by thegreekman** [Security-Onion-Solutions/securityonion#10270] [Unsupported Versions]
Hello, what file do I need to modify to retain more data for my elastic search indices in kibana? If I go back past 2 months, I see 5 events when there should be millions.

**Comment by cm-ops** [comment]
In `/opt/so/saltstack/local/pillar/global.sls` you can edit `close` and `delete` days. But in addition, if you have a basic install, you would also need to take into account log size limit in the searchnode's minion pillar - `/opt/so/saltstack/local/pillar/minions/`. If you have the disk space available the ES indices will be deleted at the the delete days in the global.sls, if not, they will get deleted at the log size limit in the minion pillar.

**Comment by thegreekman** [comment]
Thanks for the reply
